pkg/ccl/cliccl/load.go

```diff
@@ -122,6 +135,55 @@ func runLoadShowSummary(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func runLoadShowIncremental(cmd *cobra.Command, args []string) error {
+
+	path := args[0]
+	if !strings.Contains(path, "://") {
+		path = cloudimpl.MakeLocalStorageURI(path)
+	}
+	ctx := context.Background()
+	store, err := cloudimpl.ExternalStorageFromURI(ctx, path, base.ExternalIODirConfig{},
+		cluster.NoSettings, newBlobFactory, security.RootUserName(), nil /*Internal Executor*/, nil /*kvDB*/)
+	if err != nil {
+		return err
+	}
+
+	incPaths, err := backupccl.FindPriorBackupLocations(ctx, store)
+	if err != nil {
+		if errors.Is(err, cloudimpl.ErrListingUnsupported) {
+			// If we do not support listing, we have to just assume there are none
+			// and show the specified base.
+			log.Warningf(ctx, "storage sink %T does not support listing, only resolving the base backup", store)
```

Comment:
Can we actually log like this? Do you know where these logs go? I think we'll probably want to surface this more in the CLI tool (as opposed to when we were taking the backup). Perhaps we could surface this as an error to the user since this command won't work against these stores?